ocs/rook.py

```diff
@@ -204,3 +233,155 @@ def create_cephblockpool(
             raise Exception(err)
 
         return _rc
+
+    def create_cephfs(self, **kwargs):
+        """
+        A function to invoke cephfs creation via CephFS
+
+        Args:
+            **kwargs: Parameter which user want to override against defaults
+                like name, mds count etc. Keys should match as per fs resource
+                yaml.
+
+        Returns:
+            bool: True if fs healthy else False
+        """
+        if self.cephfs:
+            logger.info(f"Cephfs {self.cephfs.name} already exists")
+            return self.cephfs.health_ok()
+        self.cephfs = CephFS(self.namespace, **kwargs)
+        assert self.cephfs.health_ok()
+        return True
+
+
+class CephFS(object):
```

Comment:
According to https://docs.google.com/drawings/d/1SUbmE-TITkgWNruiKcFwrl52ULEuO_rDUtY4guIz2UY/edit
This class should inherit KubernetesVolume